### PR TITLE
anemic domain model

### PR DESCRIPTION
**Pull request by rpierlot**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

File: _posts/en/2020-10-13-anemic-domain.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    public function getContent(): string;
+    public function getCreatedAt(): string;
+    public function getUpdatedAt(): string;
+}
+```
+
+With a richer domain model, your service could look a bit like this:
+
+```php
+//class ArticleService
```
Comment (lepiaf) [inline _posts/en/2020-10-13-anemic-domain.md:194]:
```suggestion
class ArticleService
```

**Comment by ElevenWilson** [comment]
1 words might be misspelled, please check them:

**Review by ElevenTom — APPROVED** [review]

**Pull request merged**